lib/archive/diff_test.go

```diff
@@ -264,6 +264,8 @@ func TestDiffNoData(t *testing.T) {
 }
 
 func TestDiffFilterSpec(t *testing.T) {
+	t.Skip()
```

Comment:
This should all be fixed along with the final updates to the filterspec - updated Skip comment to note why it's skipped.